docs: glob support alias path (#7204)
vitejs/vite: docs/guide/features.md

Note that:

- This is a Vite-only feature and is not a web or ES standard.

> - The glob patterns are treated like import specifiers: they must be either relative (start with `./`) or absolute (start with `/`, resolved relative to project root) or alias path (alias path define in [Vite config](/config/#resolve-alias)).

Reviewer: ```suggestion
- The glob patterns are treated like import specifiers: they must be either relative (start with `./`) or absolute (start with `/`, resolved relative to project root) or an alias path (see [`resolve.alias` option](/config/#resolve-alias)).
```

Minor nitpick 😬

Author: nice!
